BootloaderCorePkg/Tools/PrepareBuildComponentBin.py

```diff
@@ -69,7 +69,7 @@ def CopyFileList (copy_list, src_dir, sbl_dir):
 
 def GetFspCopyList (platform):
     plat_pkg = GetPlatformPkgName (platform)
-    fsp_inf  = 'Silicon/%s/FspBin/FspBin.inf' % plat_pkg
+    fsp_inf  = os.path.join(os.environ['PLT_SOURCE'], 'Silicon/%s/FspBin/FspBin.inf' % plat_pkg)
```

Comment:
This could not fix the build failure issue when moving silicon package out of SblOpen since sbl_dir is hard code. 
and in this file, there is other hard code to CFL. 
It would be better to make it a generic script, so we don't need update this script when there is a new platform open sourced.

Reply:
I did test to move the CFL board/silicon to outside of SBL tree and it did work as expected.
However, if you are using different target name other than original supported 'apl', 'cfl', then it will cause issue.   Adding support for generic target name is another scope which we can address in different patch.

Reply:
OK. let's check it in firstly. Address different file name in another patch.